packages/airnode-validator/src/config/config.ts

```diff
@@ -234,3 +234,18 @@ export const configSchema = z
   })
   .superRefine(validateSecuritySchemesReferences)
   .superRefine(validateTemplateSchemes);
+
+export type Config = SchemaType<typeof configSchema>;
+export type ApiCredentials = SchemaType<typeof apiCredentialsSchema>;
+export type NodeSettings = SchemaType<typeof nodeSettingsSchema>;
+export type Template = SchemaType<typeof templateSchema>;
+export type CloudProvider = SchemaType<typeof cloudProviderSchema>;
+export type AwsCloudProvider = SchemaType<typeof awsCloudProviderSchema>;
+export type GcpCloudProvider = SchemaType<typeof gcpCloudProviderSchema>;
+export type LocalOrCloudProvider = SchemaType<typeof localOrCloudProviderSchema>;
+export type Gateway = SchemaType<typeof gatewaySchema>;
+export type ChainOptions = SchemaType<typeof chainOptionsSchema>;
+export type ChainType = SchemaType<typeof chainTypeSchema>;
+export type ChainConfig = SchemaType<typeof chainConfigSchema>;
+export type Trigger = SchemaType<typeof triggerSchema>;
+export type Heartbeat = SchemaType<typeof heartbeatSchema>;
```

Comment:
I've exported only pieces that are used by other packages. We may want to export everything instead. LMKWYT.

Reply:
I don't think it's needed. In the worst case one can use the `SchemaType<typeof someShema>`...